server/lib/nl/utils.py

```diff
@@ -847,3 +849,9 @@ def is_percapita_relevant(sv_dcid: str) -> bool:
     if skip_sv == sv_dcid:
       return False
   return True
+
+
+def get_default_child_place_type(place: detection.Place) -> str:
+  if place.dcid == constants.EARTH_DCID:
+    return 'Country'
+  return constants.CHILD_PLACES_TYPES.get(place.place_type, 'County')
```

Comment:
nit: it would be useful to document the behavior here and why might that be the case? Why is "County" the default, for example? What if the place is a non-US country?

Reply:
Done, thanks.